system-addon/content-src/components/Base/Base.jsx

```diff
@@ -30,6 +30,7 @@ class Base extends React.PureComponent {
     // dispatched right after the store is ready.
     if (this.props.isPrerendered) {
       this.props.dispatch(ac.SendToMain({type: at.NEW_TAB_STATE_REQUEST}));
+      this.props.dispatch(ac.SendToMain({type: at.PAGE_PRERENDERED}));
```

Comment:
Any reason why we added a new action instead of adding some data to NEW_TAB_STATE_REQUEST?

Reply:
I believe NEW_TAB_STATE_REQUEST has its own semantics, correct? @k88hudson